generate/generator/generate.go

```diff
@@ -130,27 +145,51 @@ func generateVersions(edition Edition, product Product) error {
 	versions := versionSubdirectories(dir)
 
 	// for each version
-	for _, version := range versions {
-
-		dockerfile := path.Join(dir, version, "Dockerfile")
-
-		if skipGeneration.Matches(product, version) {
-			log.Printf("Skipping generation for %v %v %v", product, edition, version)
-			continue
-		}
-
-		variant := DockerfileVariant{
-			Dockerfile: dockerfile,
-			Edition:    edition,
-			Product:    product,
-			Version:    strings.TrimSuffix(version, "-staging"),
-			IsStaging:  strings.HasSuffix(version, "-staging"),
-		}
-
-		if err := generateVariant(variant); err != nil {
-			return err
-		}
-
+	for _, ver := range versions {
+
+                if skipGeneration.Matches(product, ver) {
+                        log.Printf("Skipping generation for %v %v %v", product, edition, ver)
+                        continue
+                }
+
+                //versions such as 1.1.0-forestdb_bucket is not comparable.
+                //hence strip these when checking against the constraint.
+                v1, err := version.NewVersion(strings.Split(ver, "-")[0])
+                constraints, err := version.NewConstraint("> 7.0.3")
```

Comment:
This will match eg. 7.0.4 ... Is >= available? You could do ">= 7.1.0" if so. Otherwise do something like "> 7.0.99".

Reply:
done.